packages/documentation/copy/en/handbook-v2/Narrowing.md

```diff
@@ -622,9 +622,9 @@ function getArea(shape: Shape) {
 ```
 
 Like with our first definition of `Shape`, this is still an error.
-When `radius` was optional, we got an error (only in [`strictNullChecks`](/tsconfig#strictNullChecks)) because TypeScript couldn't tell whether the property was present.
+When `radius` was optional, we got an error (only with [`strictNullChecks`](/tsconfig#strictNullChecks) enabled) because TypeScript couldn't tell whether the property was present.
```

Comment:
```suggestion
When `radius` was optional, we got an error (with [`strictNullChecks`](/tsconfig#strictNullChecks) enabled) because TypeScript couldn't tell whether the property was present.
```